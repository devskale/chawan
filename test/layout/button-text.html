<style>
#x::before { content: "test" }
</style>
<input id=x type=submit><br>
<input type=submit value=><br>
<input type=reset><br>
<input type=reset value=><br>
