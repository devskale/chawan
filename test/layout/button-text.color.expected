[38;2;255;55;56m[SUBMIT][39m
[38;2;255;55;56m[][39m
[38;2;255;55;56m[RESET][39m
[38;2;255;55;56m[][39m
